MAUI/Popup/layout-customizations.md

```diff
@@ -658,3 +658,17 @@ public partial class MainPage : ContentPage
 {% endhighlight %}
 {% endtabs %}
 
+## How to disable the overlay background in .NET MAUI Popup?
+
+If the `WidthRequest`, `HeightRequest`, `StartX`, and `StartY` properties are all set for the SfPopup, the overlay background around the Popup will not be shown by default. To prevent the overlay background from showing even if these properties are not set, set the `SfPopup.ShowOverlayAlways` property to `False`. The default value of `SfPopup.ShowOverlayAlways` is `True`.

```

Comment:
This is not correctly convey the usage. in .NET MAUI its common for default popup itself. Please change it accordingly

Reply:
screenshot would be the better illustration

Reply:
Addressed the concerns provided.